temporian/implementation/numpy/data/event_set.py

```diff
@@ -587,3 +585,76 @@ def creator(self) -> Optional[Operator]:
         created EventSets have a `None` creator.
         """
         return self.node()._creator
+
+    def _repr_html_(self) -> str:
+        """HTML representation, mainly for IPython notebooks."""
+        features = self.schema.features[: config.max_display_features]
```

Comment:
What is there is less than "max_display_features" ? (same for the other indexes)

Reply:
That is not a problem in python, it will stop at the list length.